makefile

```diff
@@ -4,6 +4,7 @@
 .SUFFIXES: .cpp .o .c .h
 
 CFLAGS = -fPIC -std=c89 -O3 -march=native -Wall -Wextra -pedantic
+CCFLAGS = -fPIC -O3 -march=native -Wall -Wextra -pedantic
```

Comment:
Note that i have closed the PR because i found a bug... I will issue a new
one later tonight.

2015-04-15 14:56 GMT+02:00 Anatol Belski <notifications@github.com>:

> In makefile
> <https://github.com/lemire/simdcomp/pull/7#discussion_r28416274>:
>
> > @@ -4,6 +4,7 @@
> >  .SUFFIXES: .cpp .o .c .h
> >
> >  CFLAGS = -fPIC -std=c89 -O3 -march=native -Wall -Wextra -pedantic
> > +CCFLAGS = -fPIC -O3 -march=native -Wall -Wextra -pedantic
>
> Ahh, all clear. I'll brobably have to extend the VS makefile as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lemire/simdcomp/pull/7/files#r28416274>.
>